pages/contacts/[id]/index.js

```diff
@@ -64,13 +57,13 @@ export default function ContactDetail() {
           <strong>Phone:</strong>
         </p>
         <StyledInfo>
-          <strong>{phone}</strong>
+          <strong>{contact.phone}</strong>
         </StyledInfo>
         <p>
           <strong>Notizen:</strong>
         </p>
         <StyledInfo>
-          <strong>{note}</strong>
+          <strong>{contact.note}</strong>
         </StyledInfo>
       </StyledContainer>
```

Comment:
Just for you to know. This part could also be a list. :)

```jsx
      <StyledContainer>
        <p>
          <strong>Phone:</strong>
        </p>
        <StyledInfo>
          <strong>{phone}</strong>
          <strong>{contact.phone}</strong>
        </StyledInfo>
        <p>
          <strong>Notizen:</strong>
        </p>
        <StyledInfo>
          <strong>{note}</strong>
          <strong>{contact.note}</strong>
        </StyledInfo>
      </StyledContainer>
```

Reply:
I appreciate your comment, but I acted this way because I knew I would have to remove the styles from the list that I don't need, and it's just more code (for me) unnecessary, especially if it's a list of ONLY 2 things.

Reply:
Yes I thought that that would be your reason. I just wanted to mention to you that a list would be also ok. 💪